app/src/main/java/org/liberty/android/fantastischmemo/common/AnyMemoDBOpenHelper.java

```diff
@@ -175,6 +185,21 @@ public void onUpgrade(SQLiteDatabase database, ConnectionSource connectionSource
             database.execSQL("alter table cards add favourite Boolean");
             database.execSQL("alter table settings add hintToggle Boolean");
         }
+
+        final String CREATE_MC_TABLE = "create table " +
+                MultipleChoiceContract.MultipleChoiceCardTable.TABLE_NAME + " (" +
+                "_id" + " integer primary key autoincrement, " +
+                MultipleChoiceContract.MultipleChoiceCardTable.COLUMN_QUESTION + " string, " +
+                MultipleChoiceContract.MultipleChoiceCardTable.COLUMN_OPTION1 + " string, " +
+                MultipleChoiceContract.MultipleChoiceCardTable.COLUMN_OPTION2 + " string, " +
+                MultipleChoiceContract.MultipleChoiceCardTable.COLUMN_OPTION3 + " string, " +
+                MultipleChoiceContract.MultipleChoiceCardTable.COLUMN_OPTION4 + " string, " +
+                MultipleChoiceContract.MultipleChoiceCardTable.COLUMN_ANSWER + " string, " + ")";
+
+        if(oldVersion <= 8){
+            database.execSQL("drop table "+ MultipleChoiceContract.MultipleChoiceCardTable.TABLE_NAME);
```

Comment:
Why do you have to drop the table every time?

Reply:
I initially had a table created already but then I had to modify it so I needed to drop the existing table lol. long story short...ill remove it